wire-webapp/.env.defaults

```diff
@@ -160,6 +160,16 @@ URL_TERMS_OF_USE_TEAMS="https://wire.com/legal/#teams"
 # Sets the host URL for the website, e.g. https://wire.com
 URL_WEBSITE_BASE="https://wire.com"
 
+# Sets paths to append to a base URL
+URL_SUBPATH_CREATE_TEAM="register/email"
+
+URL_SUBPATH_MANAGE_SERVICES="services/"
+
+URL_SUBPATH_MANAGE_TEAM="login/"
+
+URL_SUBPATH_PASSWORD_RESET="/forgot/"
```

Comment:
Those should be end user configurable as of https://wearezeta.atlassian.net/browse/WPB-5889